src/index.jsx

```diff
@@ -74,6 +87,18 @@ subscribe(APP_INIT_ERROR, (error) => {
 subscribe(APP_AUTH_INITIALIZED, () => {
   getAuthenticatedHttpClient().interceptors.response.use(responseInterceptor);
 
+  getAuthenticatedHttpClient().interceptors.request.use(async (requestConfig) => {
```

Comment:
Need to enable the WAFFLE_OVERRIDE setting in ecommerce

Reply:
This will send the waffle flags to the LMS as well as written

Reply:
waffle_override is a different setting to allow setting any flag by including a query parameter
https://waffle.readthedocs.io/en/v0.8.1/usage.html#waffle-override
i believe as written you're using the testing property which can be used for any flag if the flag value is set to unknown and the testing checkbox is checked